src/proptests.rs

```diff
@@ -22,49 +22,94 @@ impl PropTestEnv {
         let mut record_log = MultiRecordLog::open(tempdir.path()).await.unwrap();
         record_log.create_queue("q1").await.unwrap();
         record_log.create_queue("q2").await.unwrap();
-        let mut state: HashMap<&'static str, Range<usize>> = HashMap::default();
+        let mut state = HashMap::default();
         state.insert("q1", 0..0);
         state.insert("q2", 0..0);
         PropTestEnv {
             tempdir,
             record_log,
-            state: HashMap::default(),
+            state,
         }
     }
 
     pub async fn apply(&mut self, op: Operation) {
-        // match op {
-        //     Operation::PartialWrite => {
-
-        //     },
-        //     Operation::Reopen => {
-        //         self.reload().await.unwrap();
-        //     },
-        //     Operation::Append { queue } => {
-        //         self.record_log.append_record(queue, position_opt, payload)
-        //     },
-        //     Operation::RedundantAppend { queue } => {
-
-        //     },
-        //     Operation::Truncate { queue, pos } => {
-
-        //     },
-        // }
+        match op {
+            Operation::PartialWrite => {
```

Comment:
I think you can remove it. I wanted to simulate a crash on the middle of a write.